index.html

```diff
@@ -15,34 +15,32 @@ <h1 class="title" > Lykke with Weather <i class="fa-solid fa-umbrella-beach"></i
         <div class="search-container">
             <input type="text" placeholder=" " name="Location" class="search-input">
             <i class="fa-solid fa-magnifying-glass fa-lg search-icon"></i>
-            <i class="fa-solid fa-location-dot fa-2xl" style="color: #c07cf8;"></i>
+            <button class="under-location" ><i class="fa-solid fa-location-dot fa-lg" style="color: #ffffff;"></i></button>
         </div>
     </form>
     <div class="item-control">
         <aside class="weather-box">
              <div class="clouds" >
                  <i class="fa-solid fa-cloud fa-2xl" style="color: #ffffff;"></i>
-                 <i class="fa-solid fa-cloud fa-xl" style="color: #ffffff;"></i>
-                 <i class="fa-solid fa-cloud fa-lg" style="color: #ffffff;"></i>
-                 <i class="fa-solid fa-cloud fa-sm" style="color: #ffffff;"></i>
-                 <i class="fa-solid fa-cloud fa-xs" style="color: #ffffff;"></i>
              </div>
              <div class="temperature-item">
                 <div class="temperature">9
                     <div class="unit">°C </div>
                 </div>
                 <p class="condition-text"> , Overcast Clouds</p>
              </div>
-             <p class="day-and-clock" > Monday , 20:22 </p>
+            <div class="day" > Monday ,
+                 <p class="clock" > 20:22 </p>
+            </div>
         </aside>
-         <div class="wind-humidity-location" >
-             <p class="text-item-wind"><i class="fa-solid fa-wind fa-xs" style="color: #ffffff;padding-right: 12px;"></i>4.63 km/h</p>
-             <p class="text-item-humidity"><i class="fa-solid fa-water fa-xs" style="color: #ffffff;padding-right: 12px;"></i>83%</p>
-             <p class="text-item-location"><i class="fa-solid fa-location-dot fa-xs" style="color: #ffffff;padding-right: 12px;"></i>paris FR</p>
+         <div class="wind-humidity-location">
+                 <p class="item-wind"> <i class="fa-solid fa-wind fa-xs" style="color: #ffffff;margin-left: 10px;margin-top: 18px;"></i> 4.63 km/h</p>
+                 <p class="item-humidity"> <i class="fa-solid fa-water fa-xs" style="color: #ffffff;margin-left: 10px;margin-top: 18px;"></i> 83%</p>
+                 <p class="item-location"> <i class="fa-solid fa-location-dot fa-xs" style="color: #ffffff;margin-left: 10px;margin-top: 18px; "></i> paris FR</p>
          </div>
     </div>
     <footer>
-        <p> <i class="fa-brands fa-github fa-sm"></i> Developed by <a href="https://github.com/mahdiehhosseinii" > MahdiehHosseinii </a> </p>
+        <p class="end" > <i class="fa-brands fa-github fa-sm"></i> Developed by <a href="https://github.com/mahdiehhosseinii" > MahdiehHosseinii </a> </p>
```

Comment:
Choose meaningful name for CSS classes to describe for what purpose the element has been created